server/helpers/resourceFieldProvider.js

```diff
@@ -0,0 +1,46 @@
+// Template/Dashboard metadata is stored inside the files
+// To read the metadata we could eval() the files, but that's dangerous.
+// Instead, we use regular expressions to pull them out
+// Assumes that:
+// * the metadata comes before config information
+// * fields in the file look like fieldname: "string" or fieldname: 'string'
+// * or, special case, html: `string`
+
+const fields = {
+  id: /\s*id:\s*("|')(.*)("|')/,
+  name: /\s*name:\s*("|')(.*)("|')/,
+  description: /\s*description:\s*("|')(.*)("|')/,
+  icon: /\s*icon:\s*("|')(.*)("|')/,
+  logo: /\s*logo:\s*("|')(.*)("|')/,
+  url: /\s*url:\s*("|')(.*)("|')/,
+  preview: /\s*preview:\s*("|')(.*)("|')/,
+  category: /\s*category:\s*("|')(.*)("|')/,
+  html: /\s*html:\s*(`)([\s\S]*?)(`)/gm,
+  featured: /\s*featured:\s*(true|false)/,
+  connections: /\s*connections:( )([\S\s]*)(,)\s*layout:/
+}
+
+const getField = (fieldName, text) => {
+  regExp = fields[fieldName];
```

Comment:
You should run the project and look at create-react-all linting errors.
This should be:
`let regExp = ...`
Or better:
`let fieldValueRegRxp = ...`